src/fuse/mount.rs

```diff
@@ -369,7 +384,9 @@ mod param {
 
     pub fn copy_slice<T: Copy>(from: &[T], to: &mut [T]) {
         debug_assert!(to.len() >= from.len());
-        to[..from.len()].copy_from_slice(from);
+        to.get_mut(..from.len())
+            .unwrap_or_else(|| panic!("Indexing is out of bounds"))
```

Comment:
panic!("Indexing is out of bounds when copying slices, from slice length={}, to slice length={}", from.len(), to.len())